libjob_queue/src/environment_varlist.c

```diff
@@ -0,0 +1,52 @@
+/*
+   Copyright (C) 2017  Statoil ASA, Norway. 
+    
+   The file 'environment_varlist.c' is part of ERT - Ensemble based Reservoir Tool. 
+    
+   ERT is free software: you can redistribute it and/or modify 
+   it under the terms of the GNU General Public License as published by 
+   the Free Software Foundation, either version 3 of the License, or 
+   (at your option) any later version. 
+    
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY 
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or 
+   FITNESS FOR A PARTICULAR PURPOSE.   
+    
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html> 
+   for more details. 
+*/
+
+#include <ert/job_queue/environment_varlist.h>
+
+#include <ert/util/hash.h>
+
+struct env_varlist_struct {
+  hash_type * varlist;
+};
+
+env_varlist_type * env_varlist_alloc() {
+  env_varlist_type * list = util_malloc( sizeof * list );
+  list->varlist = hash_alloc();
+  return list;
+}
+
+void env_varlist_add(env_varlist_type * list, char * var_name, char * var_value) {
```

Comment:
The `var_name` and `var_value` arguments should `const char *`.

Reply:
The `env_varlist_add` function should also actually set the environment variable by calling the `setenv( )` function.

Reply:
Should in addition add `env_varlist_add_pathvar()` function would get the current value and then reset with `setenv()`.